magento/magento2
# Sorted composer requirements alphabetically and updated packages with…

… known security vulnerabilities.

### Description (*)
This performs some maintenance to composer files:
- it sorts all requirements alphabetically, which wasn't the case (so it violated the `config > sort-packages` setting)
- it updates packages with known security vulnerabilities to the latest versions:
    - Upgrading composer/composer (2.6.6 => 2.7.1)
    - Upgrading phpseclib/phpseclib (3.0.35 => 3.0.37)

### Related Pull Requests


### Fixed Issues (if relevant)
None

### Manual testing scenarios (*)
1. Verify that all package requirements in the `composer.json` file are in alphabetical sort order
2. Run `composer audit` and verify no package versions have known security vulnerabilities

### Questions or comments

### Contribution checklist (*)


### Resolved issues:
1. [x] resolves magento/magento2#38515: Sorted composer requirements alphabetically and updated packages with…
